src/control/security/pem.go

```diff
@@ -38,33 +38,43 @@ import (
 )
 
 const (
-	SafeKeyPerm  os.FileMode = 0400
-	SafeCertPerm os.FileMode = 0664
-	SafeDirPerm  os.FileMode = 0700
+	MaxKeyPerm  os.FileMode = 0700
+	MaxCertPerm os.FileMode = 0664
+	MaxDirPerm  os.FileMode = 0700
 )
 
-type insecureError struct {
+type improperError struct {
 	filename string
 	perms    os.FileMode
 }
 
-func (e *insecureError) Error() string {
-	return fmt.Sprintf("%s has insecure permissions set to %#o",
+func (e *improperError) Error() string {
+	return fmt.Sprintf("%s has incorrect permissions please set to %#o",
```

Comment:
For the record, when I agreed to the suggestion of using "incorrect" vs "insecure" it was in the context of keeping the previous system of checking for a specific permissions pattern. The thought was that it was less confusing for an admin to be told that the file permissions were incorrect vs insecure, because the admin may have made the file permissions more secure than expected.

Since you've modified the code to check that the file permissions do not exceed a maximum permissiveness level, we may want to keep the security terminology. We just have to think about this stuff from the perspective of someone who is starting daos for the first time and being confronted with these error messages. What helps them to a) understand what the problem is and b) how to resolve it?

What would be clearest to me, with a minimum of grammatical parsing, would be something like this:
"%s has insecure permissions %#o (suggested: %#o)"

This would then print an error on the console like so:
```
ERROR: daos_shell: Unable to load Cerificate Data: could not load key: /home/mjmac/.daosCA/certs/admin.key has insecure permissions 0666 (suggested: 0400)
```
This tells the user what the current incorrect permissions are, and suggests an acceptable resolution. Any user with basic unix/linux experience should easily understand this.

Reply:
Agreed. Fixed the error message to that.